packages/cypher-builder/src/index.ts

```diff
@@ -19,6 +19,4 @@
 
 import * as Cypher from "./Cypher";
 
-export * from "./Cypher";
```

Comment:
:thinking: 
**question:** What is the reasoning for this change?

I'm not an expect on creating package for typescript so I may be wrong here, but this export is intended to allow for javascript based exports:
```js
const Cypher = require('cypher-builder')
```

As well as destructured exports:

```ts
import { Variable } from "cypher-builder";
```

Both of which cannot be made easily (afaik) with a default export. So by exporting both we get the best of both worlds

Reply:
Yes you're right, I was having problems importing cypher builder when I first merged the changes and this was one of the things I tried to get it working as VSCode didn't like this line but I meant to put it back to how it was!